packages/api/src/external/carequality/document/process-outbound-document-query-resps.ts

```diff
@@ -334,3 +335,27 @@ function addMetriportDocRefID({
     };
   };
 }
+
+export async function updateResultDocumentReferencesWithMetriportId(
+  results: OutboundDocumentQueryResp[],
+  cxId: string,
+  patientId: string,
+  requestId: string
+): Promise<OutboundDocumentQueryResp[]> {
+  return await Promise.all(
+    results.map(async result => {
+      const updatedDocumentReferences = result.documentReference
+        ? await Promise.all(
+            result.documentReference.map(async docRef => {
+              return await addMetriportDocRefID({ cxId, patientId, requestId })(docRef);
```

Comment:
`addMetriportDocRefID` returns a function that we can store before line 345, so we execute here only that (passing docref)